### ISSUE TITLE
shader file not found: shaders/line_aa_proj.glsl

### ISSUE THREAD

**Issue by NikosSkouteris** [mapsforge/vtm#485]
I have a bug with 0.9.2 release.  The apk is about 1.7mb smaller (than with 0.9.1 release) and I get the error
**java.lang.IllegalArgumentException: shader file not found: shaders/line_aa_proj.glsl**.

My references are:
    api 'org.mapsforge:vtm:' + project.ext.vtmVersion
    api 'org.mapsforge:vtm-themes:' + project.ext.vtmVersion
    api 'org.mapsforge:vtm-android:' + project.ext.vtmVersion
    api 'org.mapsforge:vtm-android:' + project.ext.vtmVersion + ':natives-arm64-v8a'
    api 'org.mapsforge:vtm-android:' + project.ext.vtmVersion + ':natives-armeabi'
    api 'org.mapsforge:vtm-android:' + project.ext.vtmVersion + ':natives-armeabi-v7a'
    api 'org.mapsforge:vtm-android:' + project.ext.vtmVersion + ':natives-mips'
    api 'org.mapsforge:vtm-android:' + project.ext.vtmVersion + ':natives-mips64'
    api 'org.mapsforge:vtm-android:' + project.ext.vtmVersion + ':natives-x86'
    api 'org.mapsforge:vtm-android:' + project.ext.vtmVersion + ':natives-x86_64'
    api 'org.mapsforge:vtm-jts:' + project.ext.vtmVersion

**Comment by devemux86** [comment]
The VTM artifacts can be checked in [Maven repository](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.mapsforge%22), where the shaders are included in vtm-0.9.2.jar file.

Seems like Gradle cache or Android Studio with Instant Run related. Can you check suggestions in #205?

**Comment by NikosSkouteris** [comment]
Solved! After a lot of "Clean Project" commands I followed a manual procedure. I closed Android Studio and manually deleted the "build" folders of all of my modules and after openning Android Studio again everything worked fine.

I had this problem only at build (Generate Signed APK). I was getting the error after uploading and running the APK at a device. At run everything was working fine.
So, I think it was "Gradle cache" problem and not "Instant Run"

Thanks!
